CONTRIBUTING.md

```diff
@@ -1,33 +1,154 @@
+[working-group]: https://docs.google.com/document/d/1rR0qLpsjU38nRXxeich7F5QUy73RHJ90hnZiFIQ-JJ8/edit#heading=h.ar8ibc31ux6f
+[slack]: https://kubernetes.slack.com/archives/C012F2GPMTQ
 
-- [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
-- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet) - Common resources for existing developers
+There are multiple Git repositories under the [`servicebinding` GitHub organization](https://github.com/servicebinding).
 
-## Mentorship
+- [spec](https://github.com/servicebinding/spec) - The latest actively developing working copy of the spec.
+- [service-binding-controller](https://github.com/servicebinding/service-binding-controller) - The reference implementation of the spec.
+- [website](https://github.com/servicebinding/website) - The [servicebinding.io](https://servicebinding.io) website content.
+- [conformance](https://github.com/servicebinding/conformance) - The conformace test suite.
 
-- [Mentoring Initiatives](https://git.k8s.io/community/mentoring) - We have a diverse set of mentorship programs available that are always looking for volunteers!
 
-## Working Group
+If you catch an error, please let us know by opening an issue or pull request in the related GitHub repository.
 
-The Service Binding Specification for Kubernetes project is a community lead effort.
-A bi-weekly [working group call][working-group] is open to the public.
-Discussions occur here on GitHub and on the [#bindings-discuss channel in the Kubernetes Slack][slack].
+Please note we have a [code of conduct][conduct], please follow it in all your interactions with the project.
 
-[working-group]: https://docs.google.com/document/d/1rR0qLpsjU38nRXxeich7F5QUy73RHJ90hnZiFIQ-JJ8/edit#heading=h.ar8ibc31ux6f
-[slack]: https://kubernetes.slack.com/archives/C012F2GPMTQ
+Contributions to this project should conform to the [Developer Certificate of Origin][dco]. See the [next section](#sign-your-work) for more details.
+
+## Sign Your Work
+
+Contributions to this project should conform to the [Developer Certificate of Origin][dco]. You need to sign-off your git commits before sending the pull requests. The sign-off is a single line of text at the end of the commit message. The signature consists of your official name and email address. These two details should match with the name and email address used in the Git commit. All your commits needs to be signed. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. The rules are pretty simple, if you can certify the below (from
```

Comment:
```suggestion
Contributions to this project should conform to the [Developer Certificate of Origin][dco]. You need to sign-off your git commits before sending the pull requests. The sign-off is a single line of text at the end of the commit message. The signature consists of your official name and email address. These two details should match with the name and email address used in the Git commit. All your commits need to be signed. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. The rules are pretty simple, if you can certify the below (from
```